src/days-2-4-adv/player.py

```diff
@@ -1,2 +1,52 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+class Player:
+  def __init__(self, name):
+    self.name = name
+    self.inventory = []
+    self.health = 10
+    # self.time = clock.getTime()
+    self.currentRoom = 'laboratory'
+  
+  # available move directions: l, r, f, b
+  def playerMove(x):
+    room.getCurrentRoomMoveOptions()
```

Comment:
You're coming up with neat ideas of how your objects will interact with each other.